src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -95,7 +95,7 @@
    :render-workflow-details
    (fn [{:keys [props]} workflowId]
      (let [workflows (:workflows props)
-           workflowName (get-in workflows [0 "workflowEntity" "entityName"])]
+           workflowName (get-in (filterv #(= (get-in % ["workflowId"]) workflowId) workflows) [0 "workflowEntity" "entityName"])]
```

Comment:
my gut says the underlying data structure here wants to be a map of workflow id -> other details, but i don't know the context or structure of the code